### PR TITLE
About page rebuilt

### PR DESCRIPTION
**Pull request by MSDrao**

### CODE REVIEW

File: frontend/src/views/AboutView.vue [hunk 1]
```diff
@@ -1,65 +1,127 @@
...
+        {
+          name: 'Hilary McMillan',
+          designation: 'Professor & Project Lead, San Diego State University',
+          img: 'https://lh7-rt.googleusercontent.com/docsz/AD_4nXdADheKsAYX2k0qqlrcKs1K045hXWzS2HBmPrFg9CFbIpcw5TKhCq6ufU6t8T5J7t9ZLpPzfCtzNgUXnDp-4swbkik01dvRoX_lPI09jbCHpbi6eFmkHbry9agRxKI1tC9ePF8pKpCumi5EXAXIwV3iXV9T?key=dlmBhZGLWn-vOON8gSsrvg',
+          link: 'http://www.mcmillanhydrology.org/',
+        },
+        {
+          name: 'Ryoko Araki',
+          designation: 'PhD Student & Developer, San Diego State University',
+          img: 'https://lh7-rt.googleusercontent.com/docsz/AD_4nXc_qLv2wfCPet7DY4CVIUk-q-6fAn5ySN5UKXIrXs4J8_F1OObBotovTMbgGHvdTWwRKLOpsUJvF275M-61mHUtL0IFvyW15VDQJq_pyeR8E1SC0szlcGwYzqmoydj5ryHXihvSTtqhKbSgWl98HEZXPMbf?key=dlmBhZGLWn-vOON8gSsrvg',
```
Comment (devincowan) [inline frontend/src/views/AboutView.vue:123]: I get a [429](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429) response for these images:
image

Why are we hosting the in googleusercontent?
If possible, I would either embed these directly into our build, or host them on a more reliable location

**Review by devincowan — CHANGES_REQUESTED** [review]
Looks good except for the images

File: frontend/src/views/AboutView.vue (hunk 1 above)
Comment (MSDrao) [reply frontend/src/views/AboutView.vue]: Updated the code @devincowan. Please review once

File: frontend/src/views/AboutView.vue [hunk 2]
```diff
@@ -1,65 +1,129 @@
...
+          <v-row dense>
+            <v-col v-for="member in members" :key="member.name">
+              <v-card>
+                <v-img
+                  :src="member.img"
+                  class="align-end"
+                  gradient="to bottom, rgba(0,0,0,.1), rgba(0,0,0,.5)"
+                  height="350px"
+                  cover
+                >
```
Comment (devincowan) [inline frontend/src/views/AboutView.vue:53]: image
watch your use of [cover](https://vuetifyjs.com/en/components/images/#cover)

**Review by devincowan — APPROVED** [review]

File: frontend/src/views/AboutView.vue [hunk 3]
```diff
@@ -1,65 +1,132 @@
...
+      <div>
+        <h3>Useful Links:</h3>
+        <v-list>
+          <v-list-item
+            title="Interactive Hydrologic Process Taxonomy:"
+            class="mb-5">
+            <div class="link-item-description">
+              <a href="http://mcmillanhydrology.org/ProcessTaxonomy/ProcessTaxonomyDiagram.html">
+                http://mcmillanhydrology.org/ProcessTaxonomy/ProcessTaxonomyDiagram.html
+              </a>
```
Comment (devincowan) [inline frontend/src/views/AboutView.vue:85]: intentionally including the full link in the text?

**Pull request merged**